.github/workflows/link-checker.yml

```
@@ -0,0 +1,256 @@
+            echo "## 🔗 Broken External Links Report" > link-check-results/report.md
+            echo "" >> link-check-results/report.md
+            echo "The following external links were found to be broken:" >> link-check-results/report.md
+            echo "" >> link-check-results/report.md
+            
+            # Parse JSON results for broken links
+            if [ -f "link-check-results/external-links.json" ]; then
+              # Extract broken links from JSON (simplified approach)
+              grep -o '"url":"[^"]*"' link-check-results/external-links.json | \
+              grep -v '"status":"Ok"' | \
+              sed 's/"url":"//g' | sed 's/"//g' | sort | uniq >> link-check-results/broken-links.txt || true
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)